objgraph.py

```diff
@@ -578,10 +592,14 @@ def find_chain(obj, predicate, edge_func, max_depth=20, extra_ignore=()):
 def show_graph(objs, edge_func, swap_source_target,
                max_depth=3, extra_ignore=(), filter=None, too_many=10,
                highlight=None, filename=None, extra_info=None,
-               refcounts=False, shortnames=True):
+               refcounts=False, shortnames=True, output=None):
     if not isinstance(objs, (list, tuple)):
         objs = [objs]
-    if filename and filename.endswith('.dot'):
+    if filename and output:
+        raise ValueError('Cannot specify output and filename.')
```

Comment:
I'm trying to reach 100% test coverage, so I'd like error handling to be tested too.  It'll be a simple doctest (or a unittest, whichever you prefer more).

I'm also tempted to rephrase the message as "Cannot specify both output and filename.", but I'm not a native English speaker.